.circleci/config.yml

```diff
@@ -91,16 +86,27 @@ commands:
             "$CIRCLE_PROJECT_REPONAME" \
             "$CIRCLE_BUILD_URL" > syncserver/version.json
 
+  install-test-deps:
+    steps:
+      - run: 
+          name: Install cargo-nextest
+          command: curl -LsSf https://get.nexte.st/latest/linux | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
While I have nothing against the fine residents of São Tomé and Príncipe, I do wonder if we should cache copies of stuff like this to prevent side channel attacks. (This goes double for things like `rustup.rs`)